Apps/Landing/src/vite.config.ts

```diff
@@ -1,9 +1,28 @@
 import { fileURLToPath, URL } from "node:url";
 
 import vue from "@vitejs/plugin-vue";
+import dotenv from "dotenv";
 import { defineConfig } from "vite";
 import vuetify, { transformAssetUrls } from "vite-plugin-vuetify";
 
+// Load environment variables from the appropriate .env file
+dotenv.config({
+    path: fileURLToPath(
+        new URL(
+            process.env.NODE_ENV === "production"
+                ? "./.env.production"
+                : "./.env.development",
+            import.meta.url
+        )
+    ),
+});
+
+// Prepare environment variables for Vite's define option
+const envVariables = Object.keys(process.env).reduce((prev, curr) => {
+    prev[`process.env.${curr}`] = JSON.stringify(process.env[curr]);
+    return prev;
+}, {});
```

Comment:
I'm not sure if this would be necessary if our npm scripts used the `--mode development` switch instead of setting `NODE_ENV=development` ([See docs](https://vitejs.dev/guide/env-and-mode#node-env-and-modes).)  Vite automatically loads `.env.[mode]` files.

We can merge as-is and revisit once you are back @BrianMaki to see if anything can be simplified here.